### PR TITLE
docs(cfms): add world map flight plans

### PR DESCRIPTION
**Pull request by Valastiri**
## Summary
Adds instructions and some details about using the world map flight planner that shipped with MSFS.

### Location
- docs/fbw-a32nx/feature-guides/cFMS.md

Discord username (if different from GitHub): Valastiri#8902

### CODE REVIEW

File: docs/fbw-a32nx/feature-guides/cFMS.md [hunk 1]
```diff
@@ -82,31 +82,71 @@ We have introduced new features to the custom flight management system as part o
...
 
-With the information above in mind, we have provided a feature that helps sync the aircraft's flight plan back to the MSFS' built-in flight plan. This feature can be configured on the EFB and is set to `None` by default.
+#### Steps
 
-Enabling this feature should allow:
+!!! danger "Follow each step carefully!"
 
-- **IFR clearance access through the MSFS ATC**
-- **Usage of the VFR map**
+**We recommend** the flyPad flight plan sync feature is set to `None` before attempting this. *May require you to load into a flight and check that the flyPad is set correctly.* This would allow MSFS to follow your flight plan independently of our custom FMS.
```
Comment (donstim) [inline docs/fbw-a32nx/feature-guides/cFMS.md]: May be a good recommendation, but in my first test I set the sync to none after I loaded the flight plan file from the world map and hit "fly." I think you just need to make sure it is done before you start loading the flight plan into the MCDU, either manually or by using the simBrief integration.

File: docs/fbw-a32nx/feature-guides/cFMS.md [hunk 2]
```diff
@@ -82,31 +82,71 @@ We have introduced new features to the custom flight management system as part o
...
+??? info "Finding Flight Plan Sync Setting"
+    - Go to the EFB Settings and select Sim Options. [Location Here](flyPad/settings.md#sim-options).
+    - Switch the `Sync MSFS Flight Plan` setting to `None`.
+
+1. Create a flight plan from simBrief or another 3rd party flight planner.
+2. Download and save the fight plan for MSFS in the `.pln` format to your PC.
+3. Load the `.pln` file in the MSFS world map.
+4. **Make sure** to select a starting gate and destination approach from the world map's drop down menus.
+5. Load your flight.
+6. *Optional.* You can use the MCDU to perform an INIT REQ. to pull down from simBrief to have an accurate flight plan with our custom FMS while having MSFS ATC support.
```
Comment (donstim) [inline docs/fbw-a32nx/feature-guides/cFMS.md]: Should this say something like you can use our simBrief integration feature to have an accurate flight plan with our custom FMS while having MSFS ATC support. For this, ...  [then either refer them to the simBrief integration procedure and/or remind them that they need to generate an OFP in simBrief from the same flight plan that they downloaded, i.e., they should generate an OFP, then download that same flight plan in the FS2020 format.] (FYI -  that's what simBrief calls the format.)

File: docs/fbw-a32nx/feature-guides/cFMS.md [hunk 3]
```diff
@@ -82,31 +82,71 @@ We have introduced new features to the custom flight management system as part o
...
+---
+
+Loading your flight plan via this method would allow you to use the built-in MSFS ATC to obtain your IFR clearance and ATC coverage throughout your flight. Please see other 
+methods below.
+
+!!! warning "Be Aware of the Following"
+    In most cases our custom FMS may require you to select your SID or STAR manually. (This is similar to our simBrief import on the MCDU). We highly recommend you plan your entire route including arrival procedure and runway otherwise the default ATC may not perform adequately.
+
+    If your `.pln` file contains VORs they may get shuffled around and lead to inaccurate
+
```
Comment (donstim) [inline docs/fbw-a32nx/feature-guides/cFMS.md:111]: Although this allows you to use MSFS ATC with our custom FMS throughout the flight, it does not fix the MSFS ATC system. For example, do not expect MSFS ATC to be aware of altitude constraints in SIDs and STARs, not to provide descent clearance at the proper time.

**Review by ImenesFBW — APPROVED** [review]
LGTM!

**Pull request merged**